src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
Flex redundant here, block elements by default place one beloved another

Reply:
Yes, I agree with you. But I used it to give display: block instead add it to each inline element

Reply:
So, let's remove the flex container from the card block